### PR TITLE
Cask gem init

### PR DESCRIPTION
**Pull request by whmii**
thanks for the assist @tabfugnic

### CODE REVIEW

File: cask.gemspec [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Gem::Specification.new do |spec|
+  spec.name          = "cask"
+  spec.version       = Cask::VERSION
+  spec.platform      = Gem::Platform::RUBY
+  spec.authors       = ["Will H McMahan"]
+  spec.email         = ["will@thoughtbot.com"]
+
+  spec.summary       = %q{Great Stuff}
+  spec.description   = %q{Even better Stuff}
```
Comment (mike-burns) [inline cask.gemspec]: These two things should be filled in.

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Will H McMahan
```
Comment (iwz) [inline LICENSE.txt]: you might need to add ` and thoughtbot, inc.` here

File: LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (iwz) [inline LICENSE.txt]: I don't think this file needs a `.txt` extension

File: README.md [hunk 4]
```diff
@@ -1,8 +1,10 @@
 # Cask
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/cask`. To experiment with that code, run `bin/console` for an interactive prompt.
```
Comment (iwz) [inline README.md]: this can be removed, i think?

File: bin/setup [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+bundle install
+
+# Do any other automated setup that you need to do here
```
Comment (iwz) [inline bin/setup]: can this line be removed?

File: cask.gemspec (hunk 1 above)
Comment (iwz) [inline cask.gemspec]: the Summary and Description should be updated before releasing this gem

File: bin/setup (hunk 5 above)
Comment (mike-burns) [inline bin/setup]: In fact, let's simplify this file:

```sh
#!/bin/sh
bundle install
```

File: bin/console [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+require "bundler/setup"
+require "cask"
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+# (If you use this, don't forget to add pry to your Gemfile!)
+# require "pry"
+# Pry.start
```
Comment (iwz) [inline bin/console]: remove all these comments

File: cask.gemspec (hunk 1 above)
Comment (whmii) [inline cask.gemspec]: Oop

**Pull request merged**